css/common.css

```diff
@@ -66,13 +66,3 @@ select {
 h1, h2, h3, h4, h5, h6, p {
```

Comment:
It's actually recommended that you should avoid styling elements directly.

Reply:
Sure Mahesh, I will change this. Thanks!